src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -103,7 +103,8 @@
             "(VCF or BAM read group header) will be used.")
     public String EXPECTED_SAMPLE_ALIAS;
 
-    @Option(shortName="H", doc = "A file of haplotype information produced by the CheckFingerprint program.")
+    @Option(shortName="H", doc = "File that maps SNPs to blocks in high LD. See " +
```

Comment:
"maps" is a misleading word to use here. 

The file "lists a sets of SNPs, optionally arranged in high-LD blocks, to be used for fingerprinting."